ci-operator/step-registry/openshift/e2e/gcp/private/openshift-e2e-gcp-private-workflow.yaml

```diff
@@ -0,0 +1,20 @@
+workflow:
+  as: openshift-e2e-gcp-private
+  steps:
+    env:
+      PUBLISH: Internal
+    pre:
+    - ref: gcp-provision-vpc
+    - ref: ignition-bastionhost
+    - ref: gcp-provision-bastionhost
+    - ref: proxy-config-generate
+    - chain: ipi-conf-gcp
+    - chain: ipi-install
+    test:
+    - ref: openshift-e2e-test
+    post:
+    - chain: ipi-gcp-post
```

Comment:
better to add a `gather-installer-boostrap-logs-in-bastion` here to collect the bootstrap logs if necessary